addons/t9n/models/message.py

```diff
@@ -11,14 +11,55 @@ class Message(models.Model):
     _description = "Localizable message"
 
     body = fields.Text(
+        "Entry to be Translated",
         help="The actual, textual content to be translated.",
     )
+    context = fields.Char(
+        help="Contect provided as `msgctxt` in the resource file.",
```

Comment:
"msgctxt" is specific to GNU gettext. I guess some other implementations have a different name for it. I'd go with a less specific formulation